src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -183,6 +183,13 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
             HelpMessage = "A hashtable which represents resource tags.")]
         public Hashtable Tag { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "ResourceId of the user assigned identity to be assigned to Application Gateway.")]
+        [ValidateNotNullOrEmpty]
+        public string UserAssignedIdentityId { get; set; }
```

Comment:
For parallelism with other cmdlets, this shoudl be UserAssignedIdentity - you could also add a parameter alias to allow both 'UserAssignedIdentity' and 'UserAssignedIdentityId'